fs/fs_zenfs.cc

```diff
@@ -522,10 +529,11 @@ IOStatus ZenFS::DeleteFileNoLock(std::string fname, const IOOptions& options,
   return s;
 }
 
-IOStatus ZenFS::NewSequentialFile(const std::string& fname,
+IOStatus ZenFS::NewSequentialFile(const std::string& filename,
                                   const FileOptions& file_opts,
                                   std::unique_ptr<FSSequentialFile>* result,
                                   IODebugContext* dbg) {
+  std::string fname = FormatPathLexically(filename);
```

Comment:
Maybe we could refactor `fname` to be something like `normalized_filename`.

Reply:
I did that in the first iteration for a couple of APIs, but it needed changing the name across the function causing a lot of unnecessary code churn, for no extra benefit, so chose this way.

Reply:
Always refactor if it cleans up the code :+1: